app/controllers/bulk_tagging/collections_controller.rb

```diff
@@ -0,0 +1,15 @@
+module BulkTagging
+  class CollectionsController < ApplicationController
```

Comment:
I thought this referred to the `document_collection` format - perhaps rename to something else (don't know what).